File: AppBox3/Apps/Kaomoji/Model/KaomojiFavorite+CoreDataProperties.m
//
//  KaomojiFavorite+CoreDataProperties.m
//  AppBox3
//
//  Created by Byeong-Kwon Kwak on 2/4/17.
//  Copyright © 2017 ALLABOUTAPPS. All rights reserved.
//

#import "KaomojiFavorite+CoreDataProperties.h"

@implementation KaomojiFavorite (CoreDataProperties)

+ (NSFetchRequest<KaomojiFavorite *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"KaomojiFavorite"];
}

@dynamic order;
@dynamic uniqueID;

@end

File: AppBox3/Apps/Kaomoji/Model/KaomojiFavorite_+CoreDataProperties.swift
//
//  KaomojiFavorite_+CoreDataProperties.swift
//  
//
//  Created by BYEONG KWON KWAK on 10/2/24.
//
//

import Foundation
import CoreData


extension KaomojiFavorite_ {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<KaomojiFavorite_> {
        return NSFetchRequest<KaomojiFavorite_>(entityName: "KaomojiFavorite_")
    }

    @NSManaged public var order: String?
    @NSManaged public var uniqueID: String?

}
